nucypher/characters/control/interfaces.py

```diff
@@ -103,6 +107,7 @@ def grant(self,
               n: int,
               expiration: maya.MayaDT,
               value: int = None,
+              first_period_reward: int = None,
```

Comment:
Yes, I think this parameter got removed

Reply:
#1459 is already merged into matster, so if this PR is currently based over the latest, and this variable is unused within this function it can be safely removed.